### ISSUE TITLE
YAML anchors/aliases not validated

### ISSUE THREAD

**Issue by postatum** [aml-org/amf#509]
Inspired by https://github.com/raml-org/raml-js-parser-2/issues/715

### Issue
Running code below should fail or report validation error because according to RAML 1.0 spec provided RAML input is invalid.

The spec says:
> This specification uses YAML 1.2 as its underlying format
https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md/#markup-language

> It is an error for an alias node to use an anchor that does not previously occur in the document.
https://yaml.org/spec/1.2/spec.html#id2786196

### AMF Version
4.0.0-SNAPSHOT

### Code
```sh
$ java -jar amf-4.0.0-SNAPSHOT.jar validate -in "RAML 1.0" -mime-in "application/yaml" -p "RAML" file://api.raml
{
  "@type": "http://www.w3.org/ns/shacl#ValidationReport",
  "http://www.w3.org/ns/shacl#conforms": true
}
```

### api.raml
```raml
#%RAML 1.0
title: test API
mediaType: application/json
resourceTypes:
  member:
    get:
      responses:
        200:
          body:
            example: <<param1>>
/resource1:
  type:
    member:
      param1: &first |
        {
          "id": "987",
          "at": "Sun, 20 Jul 2014 21:20:00 GMT",
          "toAddressId": "273hh79",
          "orderItemId": "9877292",
          "status": "scheduled",
          "droneId": "f"
        }
/resource2:
    type:
      member:
        param1: *second
```

**Comment by BlockLucas** [comment]
https://www.mulesoft.org/jira/browse/APIMF-1526

**Comment by gutee** [comment]
This is fixed and stated for 4.0.0 release
